dev-packages/application-manager/src/generator/webpack-generator.ts

```diff
@@ -104,7 +105,6 @@ module.exports = {
     },
     devtool: 'source-map',
```

Comment:
we should add `devtool` only if a mode is development

Reply:
I don't agree. Production doesn't mean errors don't happen, and without sourcemaps it's impossible to report issue with debug info.

Reply:
- Is not it default to webpack 4?
- What about cases when users not supposed to see the original code?

I don't think we can satisfy everybody here and would like to go with webpack 4 defaults. If someone wants source maps they can create a new webpack config which merges the default and enables source maps.

Reply:
Well, webpack's default is `eval` devtool, which results in not very good debugging experience (you can only see generated code). Though, it's probably good to gate on `development` in this case.

Reply:
in the dev mode we know what is the good default

Reply:
@ishitatsuyuki we disscused it with @svenefftinge, you can leave source maps as is, protection of proprietary extensions is a separate issue